packages/luvi/build.sh

```diff
@@ -30,6 +30,7 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 "
 
 termux_step_post_get_source() {
+	git fetch --depth=10 --recurse-submodules
```

Comment:
Wouldn't `--unshallow` be more appropriate here?
```suggestion
	git fetch --unshallow --recurse-submodules
```